python-setup/python-setup-suggestions.qmd

```diff
@@ -0,0 +1,154 @@
+# How to set up a Python environment in Positron
+
+When you do data science with Python, you should always be clear about which Python "environment" your code runs in. An environment consists of:
+
+1. An interpreter to run your Python code
+2. A set of packages for the interpreter to use
+
+Environments act as an extension of your code. The same Python code may return a different result—or not run at all—when it is run in a different environment. This creates an obvious problem for reproducibility, but the problem is easy to solve: be intentional about the Python environments you use, and document those environments alongside your code.
+
+The Positron IDE can help you do both. You can use it to work with two common types of environments:
+
+* **Virtual Environments** - which are Python's [built-in environments](https://docs.python.org/3/glossary.html#term-virtual-environment). A virtual environment is a subdirectory that contains a copy or a symlink to a Python interpreter, as well as a library of packages to be used by the environment.
+* **Conda environments** - which are Python environments managed by the [conda package manager](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html).
```

Comment:
I think I would either _also_ say something about how conda environment are literally stored on your computer, or remove that detail for venvs.

Reply:
I added where the conda environments are stored. [6aca341](https://github.com/garrettgman/positron-guides/pull/1/commits/6aca34101ef053bcd23a036ed6f919802997e4c1)